setuptools/independent/DepsProcessor/Program.cs

```diff
@@ -75,11 +78,19 @@ public static int Main(string[] args)
                 var graph = manager.Collect(exporter.GetDependencies(LibraryType.Package));
                 var expandedGraph = manager.Expand(graph, runtimes);
 
+                var trimmedRuntimeLibraries = context.RuntimeLibraries;
+
+                if (runtimepackagesToBeRemoved != null && runtimepackagesToBeRemoved.Count > 0)
+                {
+                    runtimepackagesToBeRemoved = runtimepackagesToBeRemoved.Select(x => x.ToLower()).ToList(); ;
```

Comment:
Please don't use ToLower.  Use ignore case when doing the comparisons.  Also, no reason to call ToList.